octavia-cli/octavia_cli/entrypoint.py

```diff
@@ -67,9 +81,32 @@ def set_context_object(ctx: click.Context, airbyte_url: str, workspace_id: str,
     default=True,
     help="Enable or disable telemetry for product improvement.",
 )
+@click.option(
+    "--api-headers",
+    "-h",
+    help="Additional header name and header value pairs to pass to octavia cli ex. Authorization: Basic dXNlcjpwYXNzd29yZA==",
+    multiple=True,
+    type=(str, str)
+)
+@click.option(
+    "--api-headers-file",
+    "-f",
+    envvar="AIRBYTE_HEADERS_FILE_PATH",
```

Comment:
I think we should create this file by default on `octavia init` and set the default path in the `.octavia` file.

Reply:
I fixed all change requests, thats only one which remains, take care of that tomorrow.